### PR TITLE
feat: Developer Guide - GUI - Navicat

### PR DESCRIPTION
**Pull request by shawn0915**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add develop/dev-guide-gui-navicat.md

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/15347, https://github.com/pingcap/docs/pull/16131
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Icemap — APPROVED** [review]
LGTM

File: develop/dev-guide-gui-navicat.md
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:1]: preview: https://pingcap-docsite-preview-git-pr-qiancai.vercel.app/zh/tidb/dev/dev-guide-gui-navicat

File: develop/dev-guide-gui-navicat.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+---
+title: 使用 Navicat 连接 TiDB
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:2]:
```suggestion
title: 使用 Navicat 连接到 TiDB
```

File: develop/dev-guide-gui-navicat.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+summary: 了解如何使用 Navicat 连接到 TiDB。
+---
+
+# 使用 Navicat 连接 TiDB
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:6]:
```suggestion
# 使用 Navicat 连接到 TiDB
```

File: develop/dev-guide-gui-navicat.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+TiDB 是一个兼容 MySQL 的数据库。[Navicat](https://www.navicat.com) 是为数据库用户提供的 GUI 工具集。本教程使用 [Navicat for MySQL](https://www.navicat.com/en/products/navicat-for-mysql) 工具连接 TiDB。
+
+> **警告**
+>
+> - 尽管由于 Navicat 兼容 MySQL，你可以使用 Navicat 连接到 TiDB，但 Navicat 并不完全支持 TiDB。由于 Navicat 将 TiDB 视为 MySQL，因此在使用过程中可能会遇到一些问题。[Navicat 用户管理兼容性](https://github.com/pingcap/tidb/issues/45154) 存在一个已知问题。更多 Navicat 和 TiDB 之间的兼容性问题，参考 [TiDB GitHub 问题页面](https://github.com/pingcap/tidb/issues?q=is%3Aissue+navicat+is%3Aopen)。
+> - 推荐使用其他官方支持 TiDB 的 GUI 工具，例如 DataGrip，DBeaver 以及 VS Code SQLTools。TiDB 完全支持的 GUI 工具的完整列表，参考 [TiDB 支持的第三方工具](/develop/dev-guide-third-party-support.md#gui)。
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:13]:
```suggestion
> - 建议使用其它 TiDB 完全支持的 GUI 工具进行连接，例如 DataGrip，DBeaver 以及 VS Code SQLTools。TiDB 完全支持的 GUI 工具的完整列表，参考 [TiDB 支持的第三方工具](/develop/dev-guide-third-party-support.md#gui)。
```

File: develop/dev-guide-gui-navicat.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+## 前置需求
+
+为了能够顺利完成本文中的操作，你需要：
+
+- [Navicat for MySQL](https://www.navicat.com/en/download/navicat-for-mysql) **16.3.2** 或以上版本。
+- 一个 Navicat for MySQL 的付费账号。
+- 准备一个 TiDB 集群。
+
+**如果你未有 TiDB 集群，可以按如下方式创建一个：**
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:29]:
```suggestion
**如果你还没有 TiDB 集群，可以按如下方式创建一个：**
```

File: develop/dev-guide-gui-navicat.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+- （推荐） 参考 [使用 TiDB Serverless 构建 TiDB 集群](/develop/dev-guide-build-cluster-in-cloud.md) 来创建一个 TiDB Cloud 集群。
+- 参考 [部署本地测试集群](/quick-start-with-tidb.md#部署本地测试集群) 或 [部署生产 TiDB 集群](/production-deployment-using-tiup.md) 来创建一个本地集群。
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:32]:
```suggestion
- （推荐方式）参考[创建 TiDB Serverless 集群](/develop/dev-guide-build-cluster-in-cloud.md)，创建一个 TiDB Cloud 集群。
- 参考[部署本地测试 TiDB 集群](/quick-start-with-tidb.md#部署本地测试集群)或[部署正式 TiDB 集群](/production-deployment-using-tiup.md)，创建一个本地集群。
```

File: develop/dev-guide-gui-navicat.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+## 连接到 TiDB
+
+根据你选择的 TiDB 部署选项连接到 TiDB 集群。
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:36]:
```suggestion
根据你选择的 TiDB 部署方式连接到 TiDB 集群。
```

File: develop/dev-guide-gui-navicat.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+
+<SimpleTab>
+<div label="TiDB Serverless">
+
+1. 在 TiDB Cloud 的 [**Clusters**](https://tidbcloud.com/console/clusters) 页面中，点击你目标集群的名字，并进入集群的 Overview 页面。
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:41]:
```suggestion
1. 在 TiDB Cloud 的 [**Clusters**](https://tidbcloud.com/console/clusters) 页面中，点击你目标集群的名字，进入集群的 **Overview** 页面。
```

File: develop/dev-guide-gui-navicat.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+    ![Navicat: configure connection SSL panel for TiDB Serverless](/media/develop/navicat-connection-config-serverless-ssl.png)
+
+8. 点击 **Test Connection** 以验证与 TiDB Serverless 集群的连接。
+
+9. 如果连接测试成功，你可以看到 **Connection Successful** 信息。点击 **Save** 完成连接配置。
+
+</div>
+<div label="TiDB Dedicated">
+
+1. 在 TiDB Cloud 的 [**Clusters**](https://tidbcloud.com/console/clusters) 页面中，点击你目标集群的名字，并进入集群的 Overview 页面。
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:83]:
```suggestion
1. 在 TiDB Cloud 的 [**Clusters**](https://tidbcloud.com/console/clusters) 页面中，点击你目标集群的名字，进入集群的 **Overview** 页面。
```

File: develop/dev-guide-gui-navicat.md [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+
+2. 点击右上角的 **Connect** 按钮，将会弹出连接对话框。
+
+3. 点击 **Allow Access from Anywhere**。
+
+    有关如何获取连接字符串的更多详细信息，参考 [TiDB Dedicated 标准连接](https://docs.pingcap.com/tidbcloud/connect-via-standard-connection)。
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:89]:
```suggestion
    有关如何获取连接字符串的更多详细信息，参考 [TiDB Dedicated 标准连接（英文）](https://docs.pingcap.com/tidbcloud/connect-via-standard-connection)。
```

File: develop/dev-guide-gui-navicat.md [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+3. 点击 **Test Connection** 以验证与本地部署 TiDB 集群的连接。
+
+4. 如果连接测试成功，你可以看到 **Connection Successful** 信息。点击 **Save** 完成连接配置。
+
+</div>
+</SimpleTab>
+
+## 下一步
+
+- 你可以继续阅读 [开发者文档](/develop/dev-guide-overview.md)，以获取更多关于 TiDB 应用开发的最佳实践。例如：[插入数据](/develop/dev-guide-insert-data.md)、[更新数据](/develop/dev-guide-update-data.md)、[删除数据](/develop/dev-guide-delete-data.md)、[单表读取](/develop/dev-guide-get-data-from-single-table.md)、[事务](/develop/dev-guide-transaction-overview.md)、[SQL 性能优化](/develop/dev-guide-optimize-sql-overview.md)等。
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:141]:
```suggestion
- 你可以继续阅读[开发者文档](/develop/dev-guide-overview.md)，以获取更多关于 TiDB 应用开发的最佳实践。例如：[插入数据](/develop/dev-guide-insert-data.md)、[更新数据](/develop/dev-guide-update-data.md)、[删除数据](/develop/dev-guide-delete-data.md)、[单表读取](/develop/dev-guide-get-data-from-single-table.md)、[事务](/develop/dev-guide-transaction-overview.md)、[SQL 性能优化](/develop/dev-guide-optimize-sql-overview.md)等。
```

File: develop/dev-guide-gui-navicat.md [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+- 如果你更倾向于参与课程进行学习，我们也提供专业的 [TiDB 开发者课程](https://cn.pingcap.com/courses-catalog/category/back-end-developer/?utm_source=docs-cn-dev-guide) 支持，并在考试后提供相应的 [资格认证](https://learn.pingcap.com/learner/certification-center)。
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:142]:
```suggestion
- 如果你更倾向于参与课程进行学习，我们也提供专业的 [TiDB 开发者课程](https://cn.pingcap.com/courses-catalog/category/back-end-developer/?utm_source=docs-cn-dev-guide) 支持，并在考试后提供相应的[资格认证](https://learn.pingcap.com/learner/certification-center)。
```

File: develop/dev-guide-gui-navicat.md [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+summary: 了解如何使用 Navicat 连接到 TiDB。
+---
+
+# 使用 Navicat 连接 TiDB
+
+TiDB 是一个兼容 MySQL 的数据库。[Navicat](https://www.navicat.com) 是为数据库用户提供的 GUI 工具集。本教程使用 [Navicat for MySQL](https://www.navicat.com/en/products/navicat-for-mysql) 工具连接 TiDB。
+
+> **警告**
+>
+> - 尽管由于 Navicat 兼容 MySQL，你可以使用 Navicat 连接到 TiDB，但 Navicat 并不完全支持 TiDB。由于 Navicat 将 TiDB 视为 MySQL，因此在使用过程中可能会遇到一些问题。[Navicat 用户管理兼容性](https://github.com/pingcap/tidb/issues/45154) 存在一个已知问题。更多 Navicat 和 TiDB 之间的兼容性问题，参考 [TiDB GitHub 问题页面](https://github.com/pingcap/tidb/issues?q=is%3Aissue+navicat+is%3Aopen)。
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:12]:
```suggestion
> - 尽管由于 Navicat 兼容 MySQL，你可以使用 Navicat 连接到 TiDB，但 Navicat 并不完全支持 TiDB。由于 Navicat 将 TiDB 视为 MySQL，因此在使用过程中可能会遇到一些问题。[Navicat 用户管理兼容性](https://github.com/pingcap/tidb/issues/45154) 为一个已知的问题。更多 Navicat 和 TiDB 之间的兼容性问题，参考 [TiDB GitHub issue 页面](https://github.com/pingcap/tidb/issues?q=is%3Aissue+navicat+is%3Aopen)。
```

File: develop/dev-guide-gui-navicat.md [hunk 13]
```diff
@@ -0,0 +1,146 @@
...
+
+    - **Connection Name**：为该连接指定一个有意义的名称。
+    - **Host**：输入从 TiDB Cloud 连接对话框中的得到的 `HOST` 参数。
+    - **Port**：输入从 TiDB Cloud 连接对话框中的得到的 `PORT` 参数。
+    - **User Name**：输入从 TiDB Cloud 连接对话框中的得到的 `USERNAME` 参数。
+    - **Password**：输入 TiDB Serverless 集群的密码。
+
+    ![Navicat: configure connection general panel for TiDB Serverless](/media/develop/navicat-connection-config-serverless-general.png)
+
+7. 点击 **SSL** 标签，选择 **Use SSL**，**Use authentication** 以及 **Verify server certificate against CA** 复选框。并在 **CA Certificate** 字段中填入从 TiDB Cloud 连接对话框中获取的 `CA` 文件路径。
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:72]:
```suggestion
7. 点击 **SSL** 选项卡，选择 **Use SSL**，**Use authentication** 以及 **Verify server certificate against CA** 复选框。并在 **CA Certificate** 字段中填入从 TiDB Cloud 连接对话框中获取的 `CA` 文件路径。
```

File: develop/dev-guide-gui-navicat.md [hunk 14]
```diff
@@ -0,0 +1,146 @@
...
+
+    - **Connection Name**：为该连接指定一个有意义的名称。
+    - **Host**: 输入从 TiDB Cloud 连接对话框中的得到的 `HOST` 参数。
+    - **Port**：输入从 TiDB Cloud 连接对话框中的得到的 `PORT` 参数。
+    - **User Name**: 输入从 TiDB Cloud 连接对话框中的得到的 `USERNAME` 参数。
+    - **Password**：输入 TiDB Dedicated 集群的密码。
+
+    ![Navicat: configure connection general panel for TiDB Dedicated](/media/develop/navicat-connection-config-dedicated-general.png)
+
+7. 点击 **SSL** 标签，选择 **Use SSL**，**Use authentication** 以及 **Verify server certificate against CA** 复选框。然后，在 **CA Certificate** 字段中选择第 4 步下载的 CA 文件。
```
Comment (qiancai) [inline develop/dev-guide-gui-navicat.md:107]:
```suggestion
7. 点击 **SSL** 选项卡，选择 **Use SSL**，**Use authentication** 以及 **Verify server certificate against CA** 复选框。然后，在 **CA Certificate** 字段中选择第 4 步下载的 CA 文件。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**